### ISSUE TITLE
Could the controller and router use the computed, well-known, edge trust bundle instead of the ctrl plane trust bundle?

### ISSUE THREAD

**Issue by qrkourier** [openziti/ziti#996]
The ctrl plane identity's CA trust bundle configured for the controller in `.identity.ca` must be composed before running the controller. It must contain the CA cert of the issuer of the ctrl plane server cert and the edge signer CA cert. The controller uses the bundle (not a trust "chain" per se, but rather a collection of discrete trust chains) to verify the router's client certificate.

Differently, edge SDK clients fetch the edge well-known trust bundle computed by the controller and served in the edge client API from `/.well-known/est/cacerts` as a PKCS7 trust store.

Shouldn't the controller use this same computed trust bundle to verify the router's client certificates which are always issued by the same edge signer CA?

Using the computed trust bundle instead of an externally-composed trust bundle would eliminate the operational hazard of failing to include all the necessary certificates in the trust bundle known by `.identity.ca`. This is possible because the only expected entries in this trust bundle are issuer certs that are elsewhere declared in the same controller configuration.

In summary, I propose that the controller responding to a router contacting the ctrl plane endpoint during TLS negotiation uses the same computed trust bundle as edge SDK clients instead of `.identity.ca`. This allows the meaning of `.identity.ca` to change from an externally-composed trust bundle to merely the singular trust chain of the issuer of the ctrl plane identity's client and server certificates.

**Comment by dovholuknf** [comment]
I think this is a good candidate to disucss at the March monthly forum if there are no other topics. We (I) skipped this month's forum, but I plan to post on discourse this week. I'll try to remember this for the forum

**Comment by qrkourier** [comment]
I'll try to remember to bring it up too. From the [ctrl config ref about the signingCert field](https://docs.openziti.io/docs/reference/configuration/controller#signingcert) it looks like `.identity.ca` is only part of the trust bundle that's shipped to edge identities during enrollment, so they might not be interchangeable after all.

**Comment by qrkourier** [comment]
redundant to https://github.com/openziti/ziti-doc/issues/151
